src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -1999,10 +1999,10 @@ static void M()
             var compilation = CreateCompilationWithMscorlib(text);
             var tree = compilation.SyntaxTrees.Single();
             var model = compilation.GetSemanticModel(tree);
-            var position = text.IndexOf("{", text.IndexOf("a)"));
+            var position = text.IndexOf('{', text.IndexOf("a)", StringComparison.Ordinal));
```

Comment:
Yes, but only for an isolated scenario. I didn't measure a real contribution to compiler or test performance.

Also, when StringComparison.Ordinal is explicitly provided instead of using the default behavior (i.e. current culture), it ~2.8 times faster.